firebase-installations/src/test/java/com/google/firebase/installations/FirebaseInstallationsTest.java

```diff
@@ -510,6 +513,90 @@ public void testGetId_multipleCalls_sameFIDReturned() throws Exception {
     assertTrue("the entry isn't doesn't have a registered fid: " + entry, entry.isRegistered());
   }
 
+  @Test
+  public void testGetId_expiredAuthTokenUncheckedException_statusUpdated() throws Exception {
+    // Start with a registered FID
+    persistedInstallation.insertOrUpdatePersistedInstallationEntry(
+        PersistedInstallationEntry.INSTANCE.withRegisteredFid(
+            TEST_FID_1,
+            TEST_REFRESH_TOKEN,
+            utils.currentTimeInSecs(),
+            TEST_AUTH_TOKEN,
+            TEST_TOKEN_EXPIRATION_TIMESTAMP));
+
+    // Move the time forward by the token expiration time.
+    fakeClock.advanceTimeBySeconds(TEST_TOKEN_EXPIRATION_TIMESTAMP);
+
+    // Mocking unchecked exception on FIS generateAuthToken
+    when(mockBackend.generateAuthToken(anyString(), anyString(), anyString(), anyString()))
+        .thenThrow(new IOException());
+
+    TestOnCompleteListener<String> onCompleteListener = new TestOnCompleteListener<>();
+    Task<String> getIdTask = firebaseInstallations.getId();
+    getIdTask.addOnCompleteListener(executor, onCompleteListener);
+    String fid = onCompleteListener.await();
+
+    assertWithMessage("getId Task failed").that(fid).isEqualTo(TEST_FID_1);
+
+    // Waiting for Task that generates auth token with the FIS Servers
+    executor.awaitTermination(500, TimeUnit.MILLISECONDS);
+
+    // Validate that registration status is still REGISTER
+    PersistedInstallationEntry entry = persistedInstallation.readPersistedInstallationEntryValue();
```

Comment:
line too long

Reply:
ran googleJavaFormat on this file and it seems correctly formatted as per the rules. Tried splitting the line but that throws lint errors.

Reply:
Now, it looks fine too. Sorry about it.